python/lsst/ts/Dome/dome_csc.py

```diff
@@ -12,6 +14,10 @@
 
 _LOCAL_HOST = "127.0.0.1"
 _TIMEOUT = 20  # timeout in s to be used by this module
+_DEGREES_TO_RADIANS = math.pi / 180.0
```

Comment:
numpy provides numpy.degrees and numpy.radians. Please use those and you wan't need the constants:

```python
In [1]: import numpy                                                                                     

In [2]: numpy.degrees(180)                                                                               
Out[2]: 10313.240312354817

In [3]: numpy.radians(180)                                                                               
Out[3]: 3.141592653589793

In [4]: numpy.radians(numpy.pi)                                                                          
Out[4]: 0.05483113556160755
```

Reply:
I gently disagree. For scalars it's better to use the above or `math.degrees` and `math.radians` because numpy is slow for scalars. Personally I have a slight aversion to the functions because I feel their meaning is less obvious than using a named constant, so I usually use a named constant. But either is fine.

Reply:
If you are only using this for scalars then I suggest using the math functions or named constants instead. But if you have sequences, or a mix of sequences and scalars then I agree that numpy is the way to go.

Reply:
I did some timeing, using timeit in ipython. numpy.radians is ~10x slower than math.radians. math.radians is marginally (~2x with big dev) slower than raw multiplications. raw multiplications (1000x in a for loop) is ~36us, math.radians is ~73us, numpy.radians is ~770us.

Reply:
```python
In [40]: def rd_np(): 
    ...:     for x in range(0,1000): 
    ...:         d=np.radians(180) 
    ...:                                                                                                 

In [41]: timeit rd_np()                                                                                  
759 µs ± 3.95 µs per loop (mean ± std. dev. of 7 runs, 1000 loops each)

In [42]: def rd_math(): 
    ...:     for x in range(0,1000): 
    ...:         d=math.radians(180) 
    ...:          
    ...:                                                                                                 

In [43]: timeit rd_math()                                                                                
73.8 µs ± 115 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)

In [46]: _a = math.pi / 180.0                                                                            

In [47]: _a                                                                                              
Out[47]: 0.017453292519943295

In [48]: def rd_raw(): 
    ...:     for x in range(0,1000): 
    ...:         d=180*_a 
    ...:                                                                                                 

In [49]: timeit rd_raw()                                                                                 
46.8 µs ± 75.1 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)
```

Reply:
I prefer `math` to `numpy` when there are only scalars because `math` is built in, and I feel it makes the intent of the code clearer (since `math` only works with scalars). Also numpy scalars can be a bit surprising at times, though they do a good job of imitating standard Python numeric types, so it is rarely an issue.

I heartily agree that `numpy` is the way to go if the code uses arrays or a mix of arrays and scalars. It is rare that speed of numpy with scalars is an issue, though in time critical applications it pays to be careful. I recently ran into a serious performance issue that I traced to slow performance of `astropy.time` with scalars.